app/src/androidTest/java/org/mozilla/reference/browser/SettingsMenuItemsTest.java

```diff
@@ -0,0 +1,110 @@
+package org.mozilla.reference.browser;
+
+
+import androidx.test.espresso.ViewInteraction;
+import androidx.test.rule.ActivityTestRule;
+import androidx.test.runner.AndroidJUnit4;
+import androidx.test.filters.LargeTest;
+import android.view.View;
+import android.view.ViewGroup;
+import android.view.ViewParent;
+
+import org.hamcrest.Description;
+import org.hamcrest.Matcher;
+import org.hamcrest.TypeSafeMatcher;
+import org.hamcrest.core.IsInstanceOf;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import static androidx.test.espresso.Espresso.onView;
+import static androidx.test.espresso.action.ViewActions.*;
+import static androidx.test.espresso.assertion.ViewAssertions.*;
+import static androidx.test.espresso.matcher.ViewMatchers.*;
+import static org.hamcrest.core.AllOf.allOf;
+
+@LargeTest
+@RunWith(AndroidJUnit4.class)
+public class SettingsMenuItemsTest {
```

Comment:
Let's convert this to Kotlin. Android Studio can do it for us and I think we want Kotlin tests going forward.